### PR TITLE
Use exact version for cloneInto and exportFunction

### PR DESCRIPTION
**Pull request by danielhjacobs**
According to https://blog.mozilla.org/addons/2014/04/10/changes-to-unsafewindow-for-the-add-on-sdk/#comments, this was added in Firefox 30.

#### Summary

These newly added browser support table records were added in #22767 without a version number.

#### Test results and supporting details

Not tested, but see the below blog which seems to indicate that this was added in Firefox 30:

https://blog.mozilla.org/addons/2014/04/10/changes-to-unsafewindow-for-the-add-on-sdk/

Requesting review from @Rob--W and/or @rebloor 

#### Related issues

https://github.com/openwebdocs/project/issues/206

### CODE REVIEW

**Comment by danielhjacobs** [comment]
This needs the "KR: Real BCD" label

**Review by queengooborg — APPROVED** [review]

**Comment by Rob--W** [comment]
This data is incorrect. Please wait for review from webextensions people before merging BCD claims.

This was added in Firefox 49: https://bugzilla.mozilla.org/show_bug.cgi?id=1280482

**Comment by queengooborg** [comment]
Sorry @Rob--W, this was important to land for our real values goal.  Please feel free to open a follow-up PR and I will merge it!

**Comment by Rob--W** [comment]
I can currently not edit code (from my phone). I'll open a PR later ix nobody got to it before me.

The PR is straightforward - it would be this PR, except with 49 instead of 30.

**Comment by danielhjacobs** [comment]
Opened https://github.com/mdn/browser-compat-data/pull/24529

**Comment by danielhjacobs** [comment]
My other most recent two PRs are also meant to help with the real values goal: #24525 and #24527. With those merged and all the https://github.com/mdn/browser-compat-data/pulls?q=is%3Apr+is%3Aopen+label%3A%22KR%3A+Real+BCD+%3Akey%3A%22 PRs merged, the real values goal should be complete.

**Pull request merged**
